package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "code-for-denver",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/codefordenver/code-for-denver-site"
+  },
+  "devDependencies": {
+    "gulp": "3.8.10",
+    "gulp-watch": "4.1.0",
+    "gulp-handlebars": "3.0.1 ",
+    "gulp-wrap": "0.10.1",
+    "gulp-declare": "0.3.0",
+    "gulp-concat": "2.4.3",
+    "gulp-batch": "1.0.5"
+  }
+}
```

Comment:
Slightly unfamiliar territory here... let me know if this needs to change at all.